app/components/primer/heading_component.rb

```diff
@@ -1,23 +1,46 @@
 # frozen_string_literal: true
 
 module Primer
-  # Use the Heading component to wrap a component that will create a heading element
+  # Use Heading to communicate page organization and hierarchy.
+  #
+  # ## Usage
```

Comment:
I'd be 👍🏻 to adding a `usage` tag too 😄

Reply:
I'll come back to whether we need a `@usage` tag/section after extending some other component documentations! :)